hushline/templates/directory.html

```diff
@@ -1,188 +1,172 @@
-{% extends "base.html" %}
-
-{% block title %}Directory{% endblock %}
-
-{% block content %}
-  <h2>User Directory</h2>
-  {% if directory_verified_tab_enabled %}
-    {% if not logged_in %}
-      <p class="helper section-intro">
-        👋 Hush Line connects whistleblowers with lawyers, journalists, business
-        leaders, and more. We advise speaking to a legal professional before
-        taking any confidential information from your workplace.
-      </p>
-    {% endif %}
-    <div class="directory-tabs">
-      <ul class="tab-list" role="tablist">
-        <li role="presentation">
-          <button
-            type="button"
-            class="tab active"
-            data-tab="verified"
-            role="tab"
-            aria-selected="true"
-            aria-controls="verified"
-            id="verified-tab"
-          >
-            Verified
-          </button>
-        </li>
-        <li role="presentation">
-          <button
-            type="button"
-            class="tab"
-            data-tab="all"
-            role="tab"
-            aria-selected="false"
-            aria-controls="all"
-            id="all-tab"
-          >
-            All
-          </button>
-        </li>
-      </ul>
-    </div>
+{% extends "base.html" %} {% block title %}Directory{% endblock %} {% block
```

Comment:
It would be nice to not auto format the HTML because this is now less readable. Blocks should have nested indents and not be flattened on to one line.